### ISSUE TITLE
Expose a terminal

### ISSUE THREAD

**Issue by yyoncho** [emacs-lsp/lsp-gitpod#23]
As an alternative to GUI emacs we might add an option to run TUI emacs. This can be done either by https://githubplus.com/yudai/gotty or by https://github.com/butlerx/wetty

**Comment by TxGVNN** [comment]
Hi, I'm impressed how you implement Emacs in gitpod. And I had other idea is run Emacs on terminal by using tmate.io without browser. That is my example https://github.com/txgvnn/gitpod
Some features: 
- Send tmate session to telegram channel. I think in the future Gitpod support ssh session directly, so it will be perfect
- Store Emacs configs to `/workspace` then link `$HOME/.emacs*` to `/workspace` to help keep emacs packages, cache,..
- Run Emacs daemon

**Comment by aahutsal** [comment]
Indeed @TxGVNN this dances with custom browser and `C-n` stuff are quite annoying. Although `tmate` and `tmux` itself ain't too stable. Session crashes from time to time.
Also the network latency.
Would be ideal to run the local `gitpod` on the closest available server with appropriate hardware (GPUs for AI/ML stuff for instance).

**Comment by yyoncho** [comment]
@aahutsal for C-n issue you may try Nyxt browser with application-mode enabled

**Comment by TxGVNN** [comment]
>  I think in the future Gitpod support ssh session directly, so it will be perfect

@aahutsal As I see, Gitpod supported ssh connection.
![2022-06-16-211524_1366x768_scrot](https://user-images.githubusercontent.com/9713793/174090095-3147b624-8d24-4c02-8993-d2b4fae1b891.png)

**Comment by aahutsal** [comment]
> @aahutsal As I see, Gitpod supported ssh connection.

Yeah, I discovered it, thank you. It's a bit tricky (e.g. `ssh -XY` doesn't work, means it does not `X forwarding`). But I will apparently play with it!
